.github/workflows/builder_go_slsa3.yml

```diff
@@ -182,13 +188,20 @@ jobs:
     runs-on: ubuntu-latest
     needs: [privacy-check, builder, build-dry, rng]
     steps:
+      - name: Checkout builder repository
+        uses: slsa-framework/slsa-github-generator/.github/actions/secure-builder-checkout@main
```

Comment:
This unfortunately won't pass pre-submits because `secure-builder-checkout` only exists in this PR and not at `main` currently. We can either temporarily remove the required check in order to merge, or we'll need to add the secure-builder-checkout to the repo first before using it in the top level reusable workflows.